apps/cowswap-frontend/src/modules/tokensList/containers/SelectTokenWidget/helpers.tsx

```diff
@@ -0,0 +1,157 @@
+import { ReactNode } from 'react'
+
+import { TokenWithLogo } from '@cowprotocol/common-const'
+
+import { SelectTokenModalContentProps, SelectTokenWidgetContentProps } from './types'
+
+import { ImportListModal } from '../../pure/ImportListModal'
+import { ImportTokenModal } from '../../pure/ImportTokenModal'
+import { SelectTokenModal } from '../../pure/SelectTokenModal'
+import { LpTokenPage } from '../LpTokenPage'
+import { ManageListsAndTokens } from '../ManageListsAndTokens'
+
+const EMPTY_FAV_TOKENS: TokenWithLogo[] = []
+
+export function SelectTokenModalContent(props: SelectTokenModalContentProps): ReactNode {
```

Comment:
`SelectTokenModalContent` looks like just a wrapper on top of `SelectTokenModal`, what is a goal of this component?